include/envoy/runtime/runtime.h

```diff
@@ -210,7 +210,8 @@ class Snapshot {
    * @param key supplies the key to fetch.
    * @return const std::string& the value or empty string if the key does not exist.
```

Comment:
Function comment is out of date, it refers to old behavior instead of accounting for the new "default_value" argument.